packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -91,13 +88,14 @@ class ComponentWrapper extends React.Component<IComponentWrapperProps<IDevSiteDe
     }
 
     private getClassNames(): string {
-        const classNames: string = this.props.managedClasses.componentWrapper;
+        const classNames: string = this.props.active
+            ? `${this.props.managedClasses.componentWrapper} ${this.props.managedClasses.componentWrapper__active}`
+            : this.props.managedClasses.componentWrapper ;
```

Comment:
can we break this up once more since you're making the change here?

```
const classNames: string = this.props.managedClasses.componentWrapper;

classNames = this.props.active ? `${classNames} ${this.props.managedClasses.componentWrapper__active}` : classNames;
```